lib/language-dataset-factory.js

```diff
@@ -15,7 +15,7 @@ export default class LanguageDatasetFactory {
    */
   constructor (languageData = [LatinDataset, GreekDataset]) {
     this.sets = new Map()
-    for (let Set of languageData) {
+    for (const Set of languageData) {
```

Comment:
`const Set` - is `Set` the name of variable? or what does this line mean?

Reply:
Yes, Set is the name of the variable that references the class (i.e. the class constructor function if I'm not mistaken). `const Set` means that we declare that `Set` will always reference the same class within the loop. `Set` starts with a capital because we later use it to create an instance of the class as `new Set()` and it's a convention to have class names starting with a capital letter.

Reply:
May be I am wrong - but doesn't it have a conflict with javascript Set 
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Set
?

Reply:
Oh, that's a good catch! I think there will be no conflict here: the local Set variable will probably override the global one within the loop and it will still work unless we decide to use a real Set in there, but I think it's better to rename it to avoid confusion. Will update!

Reply:
This has been renamed.